bitcoind-rpc-test/src/test/scala/org/bitcoins/rpc/common/WalletRpcTest.scala

```diff
@@ -357,9 +357,13 @@ class WalletRpcTest extends BitcoindRpcTest {
   }
 
   it should "be able to import an address" in {
+
+    val address = Bech32Address
```

Comment:
Why have this hard coded?

Reply:
So we don't need to fetch an address from one of the clients, one less thing that can fail